datadog/dogshell/wrap.py

```diff
@@ -253,6 +253,8 @@ def parse_options(raw_args=None):
 as it should appear on your Datadog stream")
     parser.add_option('-k', '--api_key', action='store', type='string',
                       help="your DataDog API Key")
+    parser.add_option('-s', '--site', action='store', type='choice', default='us', choices=['us', 'eu'], help="The site \
```

Comment:
Hmm, for this I would have kept consistency with the agent config and gave the choices `datadoghq.eu` or `datadoghq.com`

Reply:
So your suggested change would be `default='datadoghq.com', choices=['datadoghq.com', 'datadoghq.eu']`?

Reply:
Maybe support both syntaxes?
I mean, it's long to type the long hostname on a CLI.